source/whats-new.txt

```diff
@@ -12,3 +12,250 @@ What's New
+
+  The 1.7.2 Go driver contains a bug fix for a data race that can occur between 
+  creating and checking out connections when ``minPoolSize > 0``. 
+
+New features of the 1.7 Go driver release include:
+
+- Added full support for MongoDB 5.0.
+
+- Support for the :readconcern:`"snapshot"` read concern outside of 
+  multi-document transactions for certain read operations.
+
+- Improved ``WriteException`` and ``BulkWriteException`` error messages for
+  schema validation via the ``WriteError.Details`` field.
+
+
+.. _version-1.6:
+
+What's New in 1.6
+-----------------
+
+.. important:: Upgrade to Version 1.6.2 or Higher 
+
+  The 1.6.2 Go driver contains a bug fix for a data race that can occur between 
+  creating and checking out connections when ``minPoolSize > 0``. 
+
+New features of the 1.6 Go driver release include:
+
+- Added support for the MongoDB Stable API. For more information, see the 
+  :ref:`Stable API Guide <golang-stable-api>`.
+
+- Added support for connections to any MongoDB service that runs behind a load 
+  balancer. 
+  
+- Added support for creating time series collections. For more information, see 
+  the :ref:`Time Series Collections Guide <golang-time-series>`.
+
+- Added the ``Let`` option on aggregates.
+
+
+.. _version-1.5:
+
+What's New in 1.5
+-----------------
+
+New features of the 1.5 Go driver release include: 
+
+- Added support for Azure and GCP key management services with client-side field 
+  level encryption.
+
+- Added a new errors API to detect duplicate key errors, timeouts, and network 
+  errors.
+
+- Added server monitoring to monitor changes on a MongoDB deployment. 
+
+- Added errors to prevent unexpected behavior on maps that contain multiple 
+  keys being used as a hint option, sort option, or for index creation.
+
+
+.. _version-1.4:
+
+What's New in 1.4
+-----------------
+
+New features of the 1.4 Go driver release include: 
+
+- Added support for stapled and non-stapled OCSP verification.
+
+- Added ``tlsDisableOCSPEndpointCheck=true`` URI option to disable sending HTTP 
+  requests if the OCSP responder is not reachable from the driver and there is 
+  no stapled response.
+
+- Added context to errors encountered during BSON unmarshalling.
+
+- Added proper ``Unwrap`` functions for various driver error types. 
+
+
+.. _version-1.3:
+
+What's New in 1.3
+-----------------
+
+New features of the 1.3 Go driver release include: 
+
+- Added new ``mgocompat`` package that exports a BSON registry compatible with 
+  ``globalsign/mgo/bson``, which can be used via the 
+  ``ClientOptions.SetRegistry`` method.
+
+- Added new ``RegisterTypeEncoder`` and ``RegisterHookEncoder`` methods, which 
+  replace the deprecated ``RegisterEncoder`` method. A corresponding change has 
+  been made to replace ``RegisterDecoder``.
+
+
+.. _version-1.2:
+
+What's New in 1.2
+-----------------
+
+New features of the 1.2 Go driver release include: 
+
+- Added GA support for client-side field level encryption.
```

Comment:
Q: What's GA? Probably worth a parenthetical like we did with KMIP earlier in the page.

Reply:
GA is General Availability, so it's saying that the v1.2 release made CSFLE publicly available. 

Opting to just take out "GA" since I don't think that adds any more necessary context outside of just saying that the release "Added support for CSFLE."